core/consensus/babe/impl/block_executor.cpp

```diff
@@ -95,7 +95,7 @@ namespace kagome::consensus {
           // block
 
           // TODO(xDimon): Move timeout for request into config
-          sync_timer_->expiresAfter(std::chrono::seconds(30));
+          sync_timer_->expiresAfter(std::chrono::seconds(3600));
```

Comment:
What about this TODO?

Reply:
Oh, not. It is debugging artifact, and will be reverted